datameta/security.py

```diff
@@ -21,13 +21,29 @@
 # SOFTWARE.
 
 from pyramid.httpexceptions import HTTPFound, HTTPUnauthorized
+from datetime import datetime
+from typing import Optional
 
 from .models import User, ApiKey
 
 import bcrypt
 import logging
 log = logging.getLogger(__name__)
 
+def check_expiration(
+    expiration_datetime:Optional[datetime], 
+    datetime_to_check:Optional[datetime]=None
```

Comment:
What's the use case for not using `now()`?

Reply:
Can be remove